libraries/AC_Autorotation/AC_Autorotation.cpp

```diff
@@ -81,39 +79,30 @@ const AP_Param::GroupInfo AC_Autorotation::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("AS_ACC_MAX", 7, AC_Autorotation, _param_accel_max, FWD_SPD_CONTROLLER_MAX_ACCEL),
 
-    // @Param: BAIL_TIME
-    // @DisplayName: Bail Out Timer
-    // @Description: Time in seconds from bail out initiated to the exit of autorotation flight mode.
-    // @Units: s
-    // @Range: 0.5 4
-    // @Increment: 0.1
-    // @User: Advanced
-    AP_GROUPINFO("BAIL_TIME", 8, AC_Autorotation, _param_bail_time, AROT_BAIL_OUT_TIME),
-
     // @Param: HS_SENSOR
     // @DisplayName: Main Rotor RPM Sensor 
     // @Description: Allocate the RPM sensor instance to use for measuring head speed.  RPM1 = 0.  RPM2 = 1.
     // @Units: s
     // @Range: 0.5 3
     // @Increment: 0.1
     // @User: Advanced
-    AP_GROUPINFO("HS_SENSOR", 9, AC_Autorotation, _param_rpm_instance, 0),
+    AP_GROUPINFO("HS_SENSOR", 8, AC_Autorotation, _param_rpm_instance, 0),
```

Comment:
Changing the indexes here will result in params being loaded from eeprom wrong. Its fine to leave a gap, just leave a comment what the index 8 was used for so noone trys to use it again in the future.

Reply:
This is SITL only at the moment though, so I figured it was ok as it is not actually on anyone's eeprom yet?

Reply:
Ah, OK, as you say, breaking SITL for people is not such a big deal.